### PR TITLE
Feat: Updated plugin author, WP compatibility tag and version

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
- Updated Plugin author name
- Updated WP compatibility tag
- Version number changed
- PHPCS setup

### Screenshots
N/A

### Types of changes

Improvement (non-breaking change which adds functionality)

### How has this been tested?
N/A

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: ph-child.php [hunk 1]
```diff
@@ -1,24 +1,25 @@
 <?php
 
 /**
- * Plugin Name: ProjectHuddle Client Site
+ * Plugin Name: Project Huddle Client Site
```
[the remark this answers is not in the document]
Comment (sushmak02) [reply ph-child.php]: Done

File: ph-child.php [hunk 2]
```diff
@@ -29,204 +30,202 @@
...
+		public function __construct() {
+			if ( defined( 'PH_VERSION' ) ) {
+				add_action( 'admin_notices', [ $this, 'parent_plugin_activated_error_notice' ] );
 				return;
 			}
 
-			$this->whitelist_option_names = array(
-				'ph_child_id'           => array(
-					'description'       => __('Website project ID.', 'project-huddle'),
+			$this->whitelist_option_names = [
```
Comment (Nikschavan) [inline ph-child.php:75]: Let's not change to PHP short arrays right away, This is only PHP 5.6+. 
Not sure if currently plugin explicitly mentions if it is PHP 5.6+

If we want to make this change, we will have to do this in two parts, first raise php requirement to 5.6 and then after some time and 1-2 updates, start using PHP 5.6+ features

Comment (Nikschavan) [reply ph-child.php:75]: Correction, it's php 5.4+

Comment (Nikschavan) [reply ph-child.php:75]: Actually, looks like the plugin does mention it is PHP 5.6+ - https://wordpress.org/plugins/projecthuddle-child-site/

Comment (sushmak02) [reply ph-child.php:75]: Yes, should we keep these changes or lets wait for upcoming releases?

File: .gitignore [hunk 3]
```diff
@@ -19,3 +19,9 @@ wp-content/plugins/hello.php
 /sitemap.xml
 /sitemap.xml.gz
 
+# Ignore Composer's directories
+vendor/
+
+# Leave node_modules from git
+node_modules/
+
```
[the remark this answers is not in the document]
Comment (sushmak02) [reply .gitignore]: Done

File: ph-child.php (hunk 2 above)
Comment (Nikschavan) [reply ph-child.php:75]: @sushmak02 - Can you revert array syntax change for now? you can do this easily by adding the phpcs rule which comes with a autofixer.

I think you just need to change `Generic.Arrays.DisallowLongArraySyntax.Found` to `Generic.Arrays.DisallowShortArraySyntax.Found`

File: phpcs.xml.dist [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	<config name="testVersion" value="5.4-"/>
+
+	<rule ref="WordPress-Core" >
+		<exclude name="WordPress.PHP.StrictComparisons.LooseComparison" />
+		<exclude name="WordPress.PHP.StrictInArray.MissingTrueStrict" />
+		<exclude name="Generic.Arrays.DisallowLongArraySyntax.Found" />
+	</rule>
+	<rule ref="WordPress-Docs" />
+
+
```
Comment (Nikschavan) [inline phpcs.xml.dist:15]:
```suggestion
```

Remove empty lines

**Review by Nikschavan — APPROVED** [review]

**Pull request merged**